python-application/src/server.py

```diff
@@ -0,0 +1,30 @@
+from flask import Response, Flask, request
+import prometheus_client
+from prometheus_client.core import CollectorRegistry
+from prometheus_client import Summary, Counter, Histogram, Gauge
```

Comment:
I think you misunderstood the issue. We want to show how to use the OpenTelemetry Prometheus exporter https://github.com/open-telemetry/opentelemetry-python/tree/main/exporter/opentelemetry-exporter-prometheus, not the Prometheus python client.

Reply:
@nanfuka please use OTEL Python Prometheus exporter.